ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml

```diff
@@ -83,7 +91,7 @@ tests:
     cluster_profile: gcp
     test:
     - as: e2e-gcp-ovn-periodic
-      commands: make functests-on-ci
+      commands: IPERF3_BITRATE_OVERRIDE="300000000" make functests-on-ci
```

Comment:
how is that related to internal images?
probably need a separate commit

Reply:
plz also explain why is this needed at all
and wont it be smarter to have it in the git repo and not openshift/release?